20250412-P-feature-planning.md

```diff
@@ -0,0 +1,182 @@
+proposals are often overlooked.
+- **New Proposals:** In contrast, new proposals benefit from only a week of feedback from a limited number
+of contributors. This rushed process often results in features that may not align with larger project goals or
+provide lasting positive impact.
+- **Artificial Urgency:** Setting tentative release dates upfront and selecting priorities before designing them creates
+unnecessary urgency. This rush to consensus risks half-baked ideas being implemented and potentially impact
+credibility around release timelines as issues that might have been caught during the design process are identified
+and iterated on on-the-fly in a rush to get the feature completed.
+
+The current release cycle structure is ineffective as it forces us to race potentially incomplete ideas out the door.
+
+## Proposal
+I would like to propose several changes to our release process starting immediately.
+
+### Develop Proposals Between Releases
+We launched v1.15 on February 26th. Each of the current goals of this release span proposals introduced largely around
+and since then:
+
+| Proposal | Introduced At                                     | Proposed By |
+| -- |---------------------------------------------------| -- |
+|Actors: Bi-directional streaming | February 17                                       | Josh van Leeuwen |
+| Actors: PubSub | February 26, alternative proposal raised March 25 | Josh van Leeuwen, alternative by Whit Waldo |
+| Multi-App Workflows | March 24                                          | Josh van Leeuwen |
+| Event Streaming | March 24                                          | Cassie Coyle |
+| Workflow Re-runs | March 25, alternative proposal raised April 6     | Whit Waldo, alternative by Josh van Leeuwen |
+
+Despite being raised no more than a couple of weeks ago, many of these proposals have already gained significant traction. 
+However, implementing well-designed flagship features based on a three-week ideation and design timeline is unrealistic
+and isn't the process taken with larger software groups. It shouldn't be done here either. Furthermost, most of these
+issues have minimal community presence outside of the most-active contributors themselves.
+
+#### Proposed Changes
+Now is the time for brainstorming big-picture ideas when contributors take breaks from coding or casual discussions
+spark new inspiration.
+
+Discord frequently features requests for smaller features that could collectively form larger improvements both in terms
+of quality of life and feature additions. Even if these authors don't write original proposals themselves, we should document
+these ideas (in a place the stalebot won't hide them) and solicit community feedback to gauge traction. Regular community
+and meteor calls should highlight these new ideas with balanced commentary from hosts and direct viewers to relevant
+issues so they can chime in themselves.
```

Comment:
I think we can all agree it's important that we encourage end-users to self-report issues as GitHub issues where they don't already exist as GitHub Issues. If those issues do already exist, it is important that we encourage users to add their feedback and votes of support to those issues.

Discord is a fine place to have conversations and explore issues, but it **must** result in an output into GitHub Issues/proposals to be super effective as a means for building evidence and datapoints for future considerations.